package.json

```diff
@@ -146,6 +150,8 @@
   },
   "dependencies": {
     "await-semaphore": "^0.1.3",
+    "extract-zip": "^2.0.1",
```

Comment:
Which one are you using? this one or `yauzl`?

Reply:
extract-zip